ccsort/CMakeLists.txt

```diff
@@ -1,13 +1,7 @@
 set(ccsort_sources local.cc local_polar.cc transpert.cc cc_memcheck.cc classify.cc f_spinad.cc cache.cc build_F_RHF.cc local_magnetic.cc sort_tei.cc cphf_F.cc fock.cc get_params.cc scf_check.cc f_sort.cc cphf_B.cc idx_permute_multipass.cc d_sort.cc b_sort.cc build_B_RHF.cc build_abcd_packed.cc c_sort.cc distribute.cc sort_oei.cc idx_permute.cc e_spinad.cc file_build_multipass.cc get_moinfo.cc file_build.cc ccsort.cc d_spinad.cc file_build_presort.cc e_sort.cc idx_error.cc denom.cc sort_pert.cc domain_print.cc idx_permute_presort.cc)
-add_library(ccsort MODULE ${ccsort_sources})
-target_include_directories(ccsort SYSTEM PRIVATE ${PSI4_INCLUDE_DIRS})
-target_link_libraries(ccsort ${core_LIBRARY})
-set_target_properties(ccsort PROPERTIES
-  PREFIX ""
-  COMPILE_FLAGS "-std=c++11")
-if(APPLE)
-  set_target_properties(ccsort PROPERTIES LINK_FLAGS "-undefined dynamic_lookup")
-endif()
+
+add_psi4_plugin(ccsort ${ccsort_sources})
+target_include_directories(ccsort SYSTEM PRIVATE ${psi4_INCLUDE_DIR})
```

Comment:
The `psi4::core` target comes with psi4, pybind11, and python libraries attached, and the add plugin should be `target_link_libraries`-ing to that, so you may be able to drop it. But then Jet, too, had that set up another way. So, as you say, only experiment can decide. :-)